src/clj/witan/send/send.clj

```diff
@@ -10,9 +10,10 @@
   generate it)"
   ([config]
    (reset-send-report)
-   (-> (i/build-input-datasets (:project-dir config) (:file-inputs config) (:schema-inputs config))
-       (p/prepare-send-inputs (:transition-parameters config))
-       (r/run-send-model (:projection-parameters config)))))
+   (let [input (i/build-input-datasets (:project-dir config) (:file-inputs config) (:schema-inputs config))]
+     (-> (try (p/prepare-send-inputs input (:transition-parameters config))
+              (catch Exception e (println (clojure.string/join "\n" (i/check-if-dataset-is-valid input)))))
```

Comment:
@otfrom the exception isn't being caught here (this is the problem I had yesterday evening). `prepare-send-inputs` is failing at the first fn it trys to call (`row-maps`) and throwing a fairly meaningless exception, which is why I'm not capturing the message, but I instead want it to do `check-if-dataset-is-valid` in that instance, and then give me back a message with the problem